src/hooks/useQueryParam/index.ts

```diff
@@ -41,8 +41,21 @@ const useQueryParam = <T>(
 
```

Comment:
It would be nice to type the `param` parameter as `QueryParams` (maybe a followup ticket?)

Reply:
Done!